qa/multi_input/multi_input_client.py

```diff
@@ -55,18 +57,19 @@ def array_from_list(arrays):
     return np.stack(arrays)
 
 
-def batcher(dataset, batch_size, n_iterations=-1):
+def batcher(dataset, max_batch_size, n_iterations=-1):
```

Comment:
I would suggest creating test_utils ad some point to have the batcher() and randomly_shaped_samples() there.